package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "pretest": "npm run --silent lint",
     "test": "npm run --silent tests-only",
-    "tests-only": "mocha test/*.js --reporter spec",
+    "tests-only": "NODE_ICU_DATA=node_modules/full-icu mocha test/*.js --reporter spec",
```

Comment:
Whoops! This is a relic. No, we don't need the devDependency because we don't rely on Intl.NumberFormat for unit tests any more. I'll nix it.